examples/jest/.bazelrc

```diff
@@ -1 +1,6 @@
 import %workspace%/../../common.bazelrc
+
+# Do not build runfile forests by default. If an execution strategy relies on runfile
```

Comment:
Interesting, thanks for the links! It seems like this isn't particular to the jest example, right? Maybe we should have this in the /common.bazelrc so it gets picked up by all our tests, as well as new workspaces created with `@bazel/create` ?

Reply:
yea! that is a good Idea this will also help normal run files (nodejs_binary) to work on windows.

Reply:
done. Moved it into the common bazelrc